admin-ui/plugins/fido/components/types/StaticConfiguration.ts

```diff
@@ -0,0 +1,102 @@
+import { FormikProps } from 'formik'
+import { AppConfiguration, FormData } from '../../types'
+
+export interface StaticConfigurationFormValues {
+  authenticatorCertsFolder: string
+  mdsCertsFolder: string
+  mdsTocsFolder: string
+  checkU2fAttestations: boolean
+  unfinishedRequestExpiration: string | number
+  metadataRefreshInterval: string | number
+  serverMetadataFolder: string
+  userAutoEnrollment: boolean
+  enabledFidoAlgorithms: string[]
+  requestedParties: RequestedPartyForm[]
+}
+
+export interface RequestedPartyForm {
+  name?: string
+  domains?: string[]
+}
+
+export interface CredentialTypeOption {
+  key: string
+  value: string
+}
+
+export interface RequestedPartyOption {
+  key: string
+  value: string
+}
+
+export interface StaticConfigurationProps {
+  fidoConfiguration: {
+    fido: AppConfiguration
+  }
+  handleSubmit: (values: FormData) => void
+}
+
+// Formik types
+export type StaticConfigurationFormik = FormikProps<StaticConfigurationFormValues>
+
+// GluuInputRow component props
+export interface GluuInputRowProps {
+  label: string
+  name: string
+  type?: string
+  value: string | number
+  formik: StaticConfigurationFormik
+  lsize: number
+  rsize: number
+  showError: boolean
+  errorMessage: string | undefined
+}
+
+// GluuToggleRow component props
+export interface GluuToggleRowProps {
+  label: string
+  name: string
+  formik: StaticConfigurationFormik
+  lsize: number
+  rsize: number
+  doc_category: string
```

Comment:
Prefer camelCase for prop names (doc_category).

For consistency with the rest of the props, consider docCategory. Change only if downstream usages can be updated.

```diff
-  doc_category: string
+  docCategory: string
```

Search and update all references before applying.